support/README.md

```diff
@@ -6,7 +6,7 @@ automatic DCO signoff and pre-commit format checking.
 
 ## Usage
 
-To get started, you need only navigate to the Envoy project root and run:
+To get started, you need only to navigate to the Envoy project root and run:
```

Comment:
Let's consider simplify this, since I'm not sure "you need only to" is adding anything the reader really of this needs to know. Instead, how about "To get started, navigate to the Envoy project root and run:"

I think making it a bit simpler will make it easier to understand for everyone.